samples/Catalog/Catalog.windows.js

```diff
@@ -463,12 +469,31 @@ class PdfViewStyleScreen extends Component<{}> {
   render() {
     return (
       <View style={styles.page}>
-        <PSPDFKitView
-          ref="pdfView"
-          style={pdfStyle}
-          // The default file to open.
-          document="ms-appx:///Assets/pdf/annualReport.pdf"/>
+        <View style={styles.container}>
+          <Text style={styles.stlyeExplanation}>
```

Comment:
Reviewer, do you feel that this would be better within the README and just a simple link to from this page?